docs/rfcs/rfc-018-mathematical-operators.md

```diff
@@ -0,0 +1,126 @@
+# Mathematical operators
+
+## Problem
+As described in the issue #677, if we have to do a mathematical operation using the GraphQL library, 
+the common approach is to query the database and then perform the mutation with the final property value.
+The former approach has several downsides:
+- It requires multiple roundtrips to the database.
+- It's not concurrency safe.
+- It requires more code than necessary in the client.
+
+## Proposed Solution
+Add new input properties that solve atomic operations like:
+- `_ADD`
+- `_SUBTRACT`
+- `_MULTIPLY`
+- `_DIVIDE`
+
+For instance `counter_ADD` or `counter_SUBTRACT`.
+These properties should be available for all the following types:
+- Int
+- Float
+- BigInt
+
+### Usage Examples
+```graphql
+updateNodes(
+  where: { id: "e9bc687a-efd1-419d-b208" }
+  update: {
+    counter_ADD: 1,
+  }
+)
+```
+
+### Technical considerations
+#### Ambiguous property
+Given the following example: 
+```graphql
+updateNodes(
+  where: { id: "e9bc687a-efd1-419d-b208" }
+  update: {
+    counter: 10,
+    counter_ADD: 1,
+  }
+)
+```
+`counter` is ambiguous, and a strategy for this situation should be defined.
```

Comment:
Let's be very specific here and specify that we will throw an error.

Reply:
Done!